scalableminds/webknossos
# Suppress Warnings in Generated Files

bump scala version to 2.12.12. add silencer plugin to suppress warnings
 - from routes files
 - from html templates
 - from the too-far expanded implicit dataSource `Format[T]`

### Steps to test:
- Compile after cleaning, the above mentioned warnings should be gone
- CI run should be enough to ensure nothing broke

### Issues:
- fixes #4983
